src/cirrus/test.py

```diff
@@ -49,10 +84,21 @@ def main():
     Execute test command
     """
     opts = build_parser(sys.argv)
-    if opts.suite is not None:
-        nose_test(opts.suite)
-    else:
-        nose_test('default')
+    config = load_configuration()
+    mode = config.test_mode(opts.suite)
+    if opts.mode:
+        mode =  opts.mode
+
+    # backwards compat: default to nosetests
+    if mode is None:
+        mode = 'nosetests'
```

Comment:
You could replace these four lines with `mode = opts.mode or 'nosetests'` (assuming opts.mode defaults to None). You could also just set `'nosetests'` to be the default in the arg parser?